### PR TITLE
Credit Card Validation to prevent failed transaction to Troly

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
- WP-23 - failed payment created a second order in WP
- WP-25 - Orders marked as paid when payment has not been processed?
### What:
- `Place Order` will validate credit card before sending to Troly
  Alt text
### Refs:
* https://app.asana.com/0/939825968139797/1136667388142628
* https://app.asana.com/0/939825968139797/1136667388142630

### CODE REVIEW

File: wp99234.php [hunk 1]
```diff
@@ -852,57 +852,6 @@ function wp99234_export_order_to_subs($order_id, $posted_data) {
     WP99234()->_woocommerce->export_order($order_id);
 }
 
-/*
- * Also Export User's info when done placing order to sync shipping address.
- */
-add_action( 'woocommerce_checkout_order_processed', 'wp99234_export_order_users_account', 11, 2 );
```
Comment (atifempireone) [inline wp99234.php]: @rcdelfin why have we removed this?

File: vendor/composer/autoload_files.php [hunk 2]
```diff
@@ -6,5 +6,5 @@
 $baseDir = dirname($vendorDir);
 
 return array(
-    $vendorDir . '/cloudinary/cloudinary_php/src/Helpers.php',
+    'c15243c55ab4c20eff78d1e24be20dac' => $vendorDir . '/cloudinary/cloudinary_php/src/Helpers.php',
```
Comment (atifempireone) [inline vendor/composer/autoload_files.php]: what is this value?

File: vendor/composer/autoload_real.php [hunk 3]
```diff
@@ -23,33 +23,48 @@ public static function getLoader()
...
+        foreach ($includeFiles as $fileIdentifier => $file) {
+            composerRequire22bdfc564ad9e280d56584f580f48e46($fileIdentifier, $file);
         }
 
         return $loader;
     }
 }
 
-function composerRequire22bdfc564ad9e280d56584f580f48e46($file)
+function composerRequire22bdfc564ad9e280d56584f580f48e46($fileIdentifier, $file)
```
Comment (atifempireone) [inline vendor/composer/autoload_real.php]: trying to understand the reasoning for this function name? why the code?

File: wp99234.php (hunk 1 above)
Comment (rcdtroly) [reply wp99234.php]: @atifempireone that will execute same proceed using `woocommerce_checkout_order_processed` hook in https://github.com/EmpireOneGroup/wp-99234/pull/71/files#diff-ce63c46351ae040196c64a368769b51cR1030-R1041

File: vendor/composer/autoload_files.php (hunk 2 above)
Comment (rcdtroly) [reply vendor/composer/autoload_files.php]: @atifempireone that was generated using `composer`

File: vendor/composer/autoload_real.php (hunk 3 above)
Comment (rcdtroly) [reply vendor/composer/autoload_real.php]: @atifempireone same as above generate from `composer`

**Pull request merged**